.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Feature Request
+about: Suggest something to add to Matplotlib
+labels: New Feature
+---
+
+<!--
+Welcome! Thanks for thinking of a way to improve Matplotlib. If this solves a problem for you, then it probably solves that problem for lots of people! So the whole community will benefit from this request.
```

Comment:
please fix the line breaks.... 

I'd just shorten to `Thanks for thinking of a way to improve Matplotlib.` and leave off the rest.

Reply:
> please fix the line breaks....

Do you mean inside the paragraph or do you mean how the text is positioned relative to to the comment lines?

Reply:
Just we wrap most lines at 79 characters.  Of course not as big a deal with a markdown file, but...

Reply:
got it. Would you be open to avoiding that rule in the issue templates? As they are prose rather than code I think it makes more sense for the github issue editor/browser to handle the reflow/wrapping. For example the line wrapping at 79 characters in the current issue template feels unnatural to me:
![image](https://user-images.githubusercontent.com/10111092/89250038-ca808680-d5e1-11ea-9dde-d154c0b85bfd.png)

There are a few other places in the feature request template where I went over 79 lines, and the existing bug_report template seems to have inconsistent enforcement of this.

Reply:
I don't feel strongly about this, but others do. I think unless there is a good reason (i.e. it'll make horrible line continuations) we stick to 79 characters.